app/src/main/java/com/pixlee/pixleeandroidsdk/ui/uioptions/ProductViewFragment.kt

```diff
@@ -0,0 +1,310 @@
+                    filterOptions = PXLAlbumFilterOptions().apply {
+                        // hasProduct and hasPermission are often used together for displaying photos with tagged products and gotten the permission from their creators
+                        // if you don't see any photos after the loading is done, go to https://app.pixlee.com/app#albums/{your album id} and make sure your photos have the same filter conditions as your filterOptions.
+                        hasProduct = true
+                        hasPermission = true
+
+                        // more filter options
+                        // - hasPermission = true
+                        // - inStockOnly = true
+                        // - .. there are more. Please check README or PXLAlbumFilterOptions class for more filter options
+                    },
+                    sortOptions = PXLAlbumSortOptions().apply {
+                        sortType = PXLAlbumSortType.RECENCY
+                        descending = false
+                    }
+            )
+        }
+    }
+
+    val configuration by lazy {
+        ProductViewHolder.Configuration().apply {
+            mainTextStyle = TextStyle().apply {
+                color = Color.BLACK
+                size = 14.px
+                sizeUnit = TypedValue.COMPLEX_UNIT_PX
+                typeface = null
+            }
+            subTextStyle = TextStyle().apply {
+                color = Color.BLACK
+                size = 12.px
+                sizeUnit = TypedValue.COMPLEX_UNIT_PX
+                typeface = null
+            }
+            priceTextStyle = CurrencyTextStyle().apply {
+                isCurrencyLeading = true
+                defaultCurrency = "EUR" // or null
+                leftText = TextStyle().apply {
+                    color = Color.DKGRAY
+                    size = 24.px
+                    sizeUnit = TypedValue.COMPLEX_UNIT_PX
+                    typeface = null
+                }
+
+                rightText = TextStyle().apply {
+                    color = Color.DKGRAY
+                    size = 14.px
+                    sizeUnit = TypedValue.COMPLEX_UNIT_PX
+                    typeface = null
+                }
+            }
+        }
+    }
+
+    fun makeProducts(discountLayout: ProductViewHolder.DiscountLayout?, products: List<PXLProduct>, showBookmarkIcon: Boolean = false, showShopIcon: Boolean = false): ProductAdapter {
+        return ProductAdapter(
+                configuration = configuration.copy().apply {
+                    if (discountLayout != null)
+                        discountPriceTextStyle = ProductViewHolder.DiscountPrice(
+                                discountLayout = discountLayout,
+                                priceTextStyle = CurrencyTextStyle().apply {
+                                    isCurrencyLeading = true
+                                    defaultCurrency = "EUR" // or null
```

Comment:
should this be USD?

Reply:
That's the demonstration part of the demo. 
- app/ is the demo app. ProductViewFragment.kt is located under app/...../ directory.

This code is to show the SDK users how the SDK can be used. The SDK users can copy and paste this code into their project and change it for their needs